src/FixedDataTableCell.js

```diff
@@ -81,17 +82,47 @@ class FixedDataTableCell extends React.Component {
     /**
      * Whether touch is enabled or not.
      */
-    touchEnabled: PropTypes.bool
-  }
+    touchEnabled: PropTypes.bool,
+
+    /**
+     * Whether the cell is present in the viewport.
+     * We use this to skip render.
+     */
+    visible: PropTypes.bool,
+  };
 
   state = {
     isReorderingThisColumn: false,
     displacement: 0,
     reorderingDisplacement: 0
   }
 
+  // TODO (pradeep): Remove when merging to Beta (acts as a quick check to see if cells are mounted)
+  componentDidMount() {
+    console.log("cell mounted");
```

Comment:
cells are mounted only during initial scrolling while the buffer expands 
after that not a single cell seems to be mounted/unmounted